### PR TITLE
feat: added integrations methods

### PR DESCRIPTION
**Pull request by toymachiner62**

### CODE REVIEW

File: lib/Clients/Integrations.php [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+        }
+    }
+
+    /**
+     * Get an Integration Connection by id
+     *
+     * @param string $id                                The id of the Integration Connection to get
+     * @return Integration                              The Integration object
+     * @throws AccessTokenNotFoundException             If the client was instantiated without an Access Token
+     * @throws IntegrationConnectionNotFoundException   If Integratio nnot found
```
Comment (nrod80) [inline lib/Clients/Integrations.php:229]: Typo in `Integration not`

File: lib/Clients/Integrations.php [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+     *
+     * @param string $integrationId         The id of the Integration to create a connection for
+     * @param string $name                  A name to give to the IntegrationConnection
+     * @param array  $config                The configuration settings for the IntegrationConnection
+     * @return Integration                  The newly created IntegrationConnection
+     * @throws AccessTokenNotFoundException If the client was instantiated without an Access Token
+     * @throws InternalErrorException       If any errors creating an IntegrationConnection
+     * @throws UnauthorizedException        If unauthorized
+     */
+    public function createIntegrationConnection(string $integrationId, string $name, array $config): IntegrationConnection
```
Comment (nrod80) [inline lib/Clients/Integrations.php:263]: For consistency across SDK clients, should `$config` be renamed to `$connectionParams`?

File: lib/CatalyticLogger.php [hunk 3]
```diff
@@ -12,5 +13,8 @@ class CatalyticLogger
     public static function getLogger($clazz)
     {
         return new Logger($clazz);
+        // $logger = new Logger($clazz);
+        // $logger->pushHandler(new StreamHandler('php://stdout', Logger::DEBUG));
+        // return $logger;
```
[the remark this answers is not in the document]
Comment (toymachiner62) [reply lib/CatalyticLogger.php:18]: How are you guys doing logging in either the .net or node sdk? I have this class which gets a logger, but there's no way to actually set where to log to, or the log level, etc.

File: lib/Clients/Integrations.php (hunk 2 above)
Comment (toymachiner62) [reply lib/Clients/Integrations.php:263]: Yea I certainly can. Should the create integration method also rename config to connectionParams? I noticed on both the .net and node ones it's config.

**Review by benblair — APPROVED** [review]

File: lib/CatalyticLogger.php [hunk 4]
```diff
@@ -11,6 +12,14 @@ class CatalyticLogger
...
     public static function getLogger($clazz)
     {
+        $logLevel = getenv('CATALYTIC_LOG_LEVEL');
+
+        if ($logLevel === 'DEBUG') {
+            $logger = new Logger($clazz);
+            $logger->pushHandler(new StreamHandler('php://stdout', Logger::DEBUG));
+            return $logger;
+        }
+
```
Comment (toymachiner62) [inline lib/CatalyticLogger.php:22]: I simply added the ability to turn on debug to stdout via an env var for now.

**Pull request merged**